packages/portal/src/components/search/SearchInterface.vue

```diff
@@ -294,22 +289,53 @@
 
     computed: {
       advancedSearchQueryCount() {
-        return this.userParams?.qa ? [].concat(this.userParams?.qa).length : 0;
+        return this.qa.length;
       },
       userParams() {
         return this.$route.query;
       },
+      apiParams() {
+        const params = ['boost', 'qf', 'query', 'reusability', 'sort'].reduce((memo, field) => {
```

Comment:
Why is the sort param added?

Reply:
Because it's convenient for testing to be able to put one's own sort param into the URL.